docs/api/domains.md

```diff
@@ -135,5 +135,25 @@ const Planets = {
   }
 }
 
-repo.push(Actions.add, { name: 'earth' }) // this will add Earth
+repo.push(addPlanet, { name: 'earth' }) // this will add Earth
+```
+
+Registration is given the command associated with an action, and the
```

Comment:
hmm... I think that makes some sense, but it that change has pretty wide documentation and api implications. `intent` was _just_ renamed to `action`, which seems like now it'd need to be changed again to `command`.

I agree it needs some distinction though, and don't really have a better idea.